### DISCUSSION TITLE
Working fine on my 2018 Hawaii model #54154

### DISCUSSION THREAD

**Discussion by wyliecoyoteuk** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#126] [General]
Finally got around to installing this on my backup pump. (I have a 2020 wifi model already)
2019 Hawaii, Pump model #54154. 6 wire.
Works fine, better than the Bestway version, (and don't even mention the LayZspa app)
I have a few spare PCBs available (the early type with space for 7 resistors- I used 7 wire links instead).
I will be looking to set it up with Home assistant.
Thanks for this superb project.
Just a note, uncommenting the last 4 lines for OTA didn't work until I put a linefeed after the monitor speed line.
![IMG_20210929_110003022](https://user-images.githubusercontent.com/79256941/135249952-9c8b6595-3b90-4656-9c61-ecaa5215c28f.jpg)

**Comment by visualapproach** [comment]
> Works fine, better than the Bestway version, (and don't even mention the LayZspa app)

Yay, that's a nice statement 🥇 thank you sir!
I added your model to the list of compatible pumps.
Ha is great! Here is my HA tub page:
![image](https://user-images.githubusercontent.com/22987250/135273242-7d1fcf06-505c-4e03-ae8a-f719b418f4cb.png)

The tub icon links to the first page
![image](https://user-images.githubusercontent.com/22987250/135273729-73289715-9e50-443b-8287-e661e6020e4b.png)
